example/pythonlib/basic/1-simple/build.mill

```diff
@@ -1,27 +1,59 @@
 package build
 import mill._, pythonlib._
 
-// Docs for PythonLib
 object foo extends PythonModule {
-  object bar extends PythonModule {
-    def pythonDeps = Seq("pandas==2.2.3", "numpy==2.1.3")
+  def pythonDeps = Seq("pandas==2.2.3", "numpy==2.1.3")
+
+  object test extends PythonTests {
+    def moduleDeps = Seq(foo)
+    def mainFileName: T[String] = Task { "test.py" }
```

Comment:
What does the test module require a `mainFileName`? Shouldn't the test framework discover the files automatically?